app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java

```diff
@@ -195,19 +167,14 @@ private void prepareLanguages() {
         String languageCode = getCurrentLanguageCode();
         if (languageCode.equals("")){
             // If current language code is empty, means none selected by user yet so use phone local
-            langListPreference.setSummary(Locale.getDefault().getDisplayLanguage());
```

Comment:
I suspect this is definitely a behaviour change

Reply:
I tested it for me it seems fine. I will revert this change if needed.

Reply:
Nope, I was wrong. Works great